browser/views.py

```diff
@@ -688,6 +689,39 @@ def delete_investigator_record(request, person_id, research_id):
 
 
 @staff_member_required
+def add_person(request):
```

Comment:
why don't you reuse 'edit_person'? It's basically the same code. You just need to make some minor adjustments.